package.json

```diff
@@ -40,7 +40,7 @@
     "chalk": "^1.1.3",
     "cldr-core": "28.0.0",
     "ember-assign-polyfill": "^2.0.1",
-    "ember-cli-babel": "^6.0.0",
+    "ember-cli-babel": "^6.6.0",
```

Comment:
I confirmed this, disregard.  https://github.com/babel/ember-cli-babel/blob/master/package.json#L39